bin/alluxio-masters.sh

```diff
@@ -29,15 +31,15 @@ DEFAULT_LIBEXEC_DIR="${BIN}/../libexec"
 ALLUXIO_LIBEXEC_DIR=${ALLUXIO_LIBEXEC_DIR:-${DEFAULT_LIBEXEC_DIR}}
 . ${ALLUXIO_LIBEXEC_DIR}/alluxio-config.sh
 
-HOSTLIST=$(cat "${ALLUXIO_CONF_DIR}/masters" | sed  "s/#.*$//;/^$/d")
+HOSTLIST=($(echo $(cat "${ALLUXIO_CONF_DIR}/masters" | sed  "s/#.*$//;/^$/d")))
```

Comment:
Actually it's a small change, because we want to print the hostname of the failed node one by one later, we directly change it as an ARRAY instead of a STRING(which cannot be accessed with de ARRAY INDEX).  The detail lies in the following

Example 1(The original usage):
a1="aaa bbb ccc ddd"
for e in ${a1[@]}; do echo $e; done

It shows:
aaa
bbb
ccc
ddd

Example 2:
a2=(aaa bbb ccc ddd)
for e in ${a2[@]}; do echo $e; done

It shows: 
aaa
bbb
ccc
ddd

We can see that both of the ways can traverse the ARRAY elements, however, we can only ACCESS the element with INDEX like ${a2[2]} as "bbb"(like a hostname)

Reply:
ah that makes sense, thanks for explaining!